src/chips/LMS7002M/calibrations.cpp

```diff
@@ -1,1539 +0,0 @@
-            PUSH_GMEASUREMENT_VALUES(++index, ChipRSSI_2_dBFS(rssi));
-        }
-    }
-    else
-#endif
-    {
-        uint8_t g_rxloopb_rfe = 2;
-        Modify_SPI_Reg_bits(G_RXLOOPB_RFE, g_rxloopb_rfe);
-        rssi = GetRSSI();
-        PUSH_GMEASUREMENT_VALUES(++index, ChipRSSI_2_dBFS(rssi));
-        lime::debug("Initial gains:\tG_RXLOOPB: %2i, CG_IAMP: %2i | %2.3f dBFS", g_rxloopb_rfe, cg_iamp, ChipRSSI_2_dBFS(rssi));
-        while (rssi < target_rssi)
-        {
-            g_rxloopb_rfe += 2;
-            if (g_rxloopb_rfe > 15)
-                break;
-            Modify_SPI_Reg_bits(G_RXLOOPB_RFE, g_rxloopb_rfe);
-            rssi = GetRSSI();
-            PUSH_GMEASUREMENT_VALUES(++index, ChipRSSI_2_dBFS(rssi));
-        }
-    }
-#ifdef DRAW_GNU_PLOTS
-    DrawMeasurement(gp, gMeasurements);
-    gMeasurements.clear();
-#endif // DRAW_GNU_PLOTS
-
-    PUSH_GMEASUREMENT_VALUES(index, ChipRSSI_2_dBFS(rssi));
-    while (rssi < dBFS_2_ChipRSSI(-27))
-    {
-        cg_iamp += 2;
-        if (cg_iamp > 63 - 6)
-            break;
-        Modify_SPI_Reg_bits(CG_IAMP_TBB, cg_iamp);
-        rssi = GetRSSI();
-        PUSH_GMEASUREMENT_VALUES(++index, ChipRSSI_2_dBFS(rssi));
-    }
-
-    while (rssi < target_rssi)
-    {
-        cg_iamp += 1;
-        if (cg_iamp > 62)
-            break;
-        Modify_SPI_Reg_bits(CG_IAMP_TBB, cg_iamp);
-        rssi = GetRSSI();
-        PUSH_GMEASUREMENT_VALUES(++index, ChipRSSI_2_dBFS(rssi));
-    }
-    if (extLoopback)
-        lime::debug("Adjusted gains:\tLOSS_MAIN_TXPAD: %2i, CG_IAMP: %2i | %2.3f dbFS",
-            Get_SPI_Reg_bits(LOSS_MAIN_TXPAD_TRF),
-            Get_SPI_Reg_bits(CG_IAMP_TBB),
-            ChipRSSI_2_dBFS(rssi));
-    else
-        lime::debug("Adjusted gains: G_RXLOOPB: %2i, CG_IAMP: %2i | %2.3f dbFS",
-            Get_SPI_Reg_bits(G_RXLOOPB_RFE),
-            Get_SPI_Reg_bits(CG_IAMP_TBB),
-            ChipRSSI_2_dBFS(rssi));
-#ifdef DRAW_GNU_PLOTS
-    DrawMeasurement(gp, gMeasurements);
-    gMeasurements.clear();
-    gp.writef("%i %f\n%i %f\ne\n", 0, ChipRSSI_2_dBFS(target_rssi), index, ChipRSSI_2_dBFS(target_rssi));
-    gp.flush();
-#endif
-    const float signalLevel_dBFS = ChipRSSI_2_dBFS(rssi);
-    constexpr float expectedSignalLevel_dBFS = -30.0;
-    if (signalLevel_dBFS < expectedSignalLevel_dBFS)
-    {
-        const char* messageFormat = "Low calibration test signal level (%3.1f dBFS), expected to be more than (%3.1f dBFS)."
-                                    " Calibration results might be impacted. Try re-calibrating or adjusting the RX gains.";
-        if (signalLevel_dBFS < -50.0)
-        {
-            lime::error(messageFormat, signalLevel_dBFS, expectedSignalLevel_dBFS);
-            return MCU_LOOPBACK_SIGNAL_WEAK;
-        }
-        else
-            lime::warning(messageFormat, signalLevel_dBFS, expectedSignalLevel_dBFS);
-    }
-    return MCU_NO_ERROR;
-}
-
-uint8_t CalibrateRx(bool extLoopback, bool dcOnly)
```

Comment:
This file contains the actual calibration code, which should be executed. It shoud not be removed.